pkg/reconciler/pipelinerun/pipelinerun_test.go

```diff
@@ -1313,15 +1307,15 @@ status:
 	wantEvents := []string{
 		"Normal Succeeded All Tasks have completed executing",
 	}
-	reconciledRun, clients := prt.reconcileRun("foo", pipelineRunName, wantEvents, false)
+	reconciledRun, clients := prt.reconcileRun(namespace, pipelineRunName, wantEvents, false)
 
 	actions := clients.Pipeline.Actions()
```

Comment:
I think it would make sense to remove the code related to client Actions altogether. IMO, this tests implementation details, rather than what's visible to users.

Reply:
I think that this code should be retained,  after the refactoring, it still performs the important task of checking the create action process, which is not covered by the current verification process that only checks the creation results.

Reply:
I disagree actually-- if we want to check whether a taskrun was created, we can just query the fake client to see if it exists. In this case, we're checking that a completed pipelinerun was updated with the results of its taskruns, but I'm not sure what the "actions" should be here. Would they be 2 "get" calls, one per child? or a "list" plus an "update"? I'm not sure what the correct assertion should be, so I don't think we should test it.